sbi/analysis/conditional_density.py

```diff
@@ -185,7 +186,7 @@ def conditional_corrcoeff(
 class ConditionedMDN:
     def __init__(
         self,
-        net: nn.Module,
+        gaussianMDN: DensityEstimator,
```

Comment:
let's follow PEP 8 for variable name: `gaussian_mdn`, although I would prefer just `mdn` to keep it short (and we don't have other MDNs yet, and if we do, then we can keep this name).

Reply:
We import nflows mdn as `mdn`, so naming it mdn will overwrite this.

But I can import it as something different and then name it mdn.